src/poliastro/plotting/gabbard.py

```diff
@@ -0,0 +1,74 @@
+from astropy import units as u
+from matplotlib import pyplot as plt
+
+from poliastro.plotting.util import generate_label
+
+
+class GabbardPlotter:
+    def __init__(self, ax, dark=False):
+
+        self._ax = ax
+        if not self._ax:
+            if dark:
+                with plt.style.context("dark_background"):
+                    _, self._ax = plt.subplots(figsize=(6, 6))
+            else:
+                _, self._ax = plt.subplots(figsize=(6, 6))
+
+        self._frame = None
+
+    @staticmethod
+    def _get_orbit_properties(orbit):
+        return orbit.r_p, orbit.r_a, orbit.period
+
+    def _get_orbit_property_list(self, orbits):
+        apogees = []
+        perigees = []
+        periods = []
+
+        for orbit in orbits:
+            perigee, apogee, period = self._get_orbit_properties(orbit)
```

Comment:
We could simplify this to remove the static method:

```suggestion
            perigee, apogee, period = orbit.r_p, orbit.r_a, orbit.period
```